frontend/src/components/Hero.tsx

```diff
@@ -1,71 +1,19 @@
- import React from "react";
+"use client";
```

Comment:
Pure Hero.tsx ko CSR karne ki need nahi hai, just do it on the child component required